.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+
+<!--- Describe your changes in detail -->
+
+Fixes # (issue)
+
+## Checklist:
+
+- [ ] The commit message follows our guidelines
```

Comment:
Do we have guidelines for this? 😅

I couldn't find any. We don't have a contributing guide yet, which I'd assume would contain this. Unless there's one, I'd remove this line. And if we've one, we should link to it.